includes/class-omise-capabilities.php

```diff
@@ -251,4 +251,20 @@ public function getInstallmentMinLimit()
 	{
 		return $this->capabilities['limits']['installment_amount']['min'];
 	}
+
+	/**
+	 * Retrieves details of TrueMoney from capabilities.
+	 *
+	 * @param string $sourceType
+	 */
+	public function get_truemoney_backend($sourceType)
```

Comment:
@aashishgurung this files goes back and forth re: function naming conventions (e.g. `getInstallmentMinLimit` v.s. `get_truemoney_backend`). Which should settle on?

Reply:
Yes, I noticed that as well and I am part of the mess. I started using snake case i.e `get_truemoney_backend` because that's the official Wordpress naming convention and how other function in Wordpress and WooCommerce are named.

Reply:
Great, thanks. Can we document this convention somewhere? README?